scriptmodules/emulators/mupen64plus.sh

```diff
@@ -321,6 +321,10 @@ function configure_mupen64plus() {
     # on the rp-dist file. This preserves any user configs from modification and allows us to have
     # a default config for reference
     if [[ -f "$config" ]]; then
+        # patch existing mupen64plus config because Gliden64 overwrites GlideN64 user configs
+        # if configVersion is wrong 
+        grep -q "configVersion" "$config" && iniConfig " = " "" "$config" && iniSet "configVersion" "$(cat $md_inst/share/mupen64plus/GLideN64_config_version.ini)"
```

Comment:
This isn't a good one-liner imho. It would be more readable to split it up and have inside an `if`

Also duplicating the path - keeping the config in a variable would be better.

Also note you can do `$(<FILE)` instead of cat.